Twig/SecurityExtensions.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\UserBundle\Twig;
+
+final class SecurityExtensions extends \Twig_Extension
+{
+    /**
+     * @return array
+     */
+    public function getFunctions()
+    {
+        return array(
+            'roleInTable' => new \Twig_Function_Method($this, 'roleInTable'),
+            'isReadOnly' => new \Twig_Function_Method($this, 'isReadOnly'),
```

Comment:
These functions are specific to the security section, but their names are too generic (and not very descriptive).
Also I think we use snake case for functions in sonata.